src/app/proposals/draft/utils/createSnapshot.ts

```diff
@@ -66,7 +66,7 @@ export async function createSnapshot({
 }) {
   const tenant = Tenant.current();
   const walletClient = getWalletClient(tenant.contracts.token.chain.id);
-  const publicClient = getPublicClient(tenant.contracts.token.chain.id);
+  const publicClient = getPublicClient(1);
```

Comment:
needs to be 1, even on testnet it tries to read current block so we can't use testnet block